### PR TITLE
fix(email): honor Loops LMX attribute ranges

### PR DESCRIPTION
**Pull request by danielkov**
## Summary

- translate compact LMX labels and column gaps to Loops-supported minimum values across all nine templates and the reusable base
- validate provider-supported `Columns.gap` and `Paragraph.fontSize` ranges locally across every `.lmx` file
- document the constrained MJML-to-LMX translation in the transactional-email skill

## Why

The first merge-driven dev reconciliation rejected the repository LMX before publish because Loops requires column gaps from 12–150 and paragraph font sizes from 12–64. This fixes every affected template and makes the same values fail `--validate-only` before merge.

## Validation

- `mise exec -- go run ./server/cmd/sync-loops-email-templates --validate-only`
- `mise exec -- go test ./server/internal/email/... ./server/cmd/sync-loops-email-templates`
- `mise lint:server`
- `mise run skills:sync`
- fresh-context skill evaluation: ship, 9.7/10
- `git diff --check`

## Visual QA

The canonical MJML design and copy are unchanged; this patch only adapts the production LMX fallback to provider minimums. Desktop/mobile previews for all nine templates remain attached to #5238: https://github.com/speakeasy-api/gram/pull/5238#issuecomment-5283345472

Production-render QA remains gated on the authorized release reconciliation; no manual Loops mutation or test-send was performed.

---
## Summary by cubic
Honors Loops’ LMX attribute ranges to prevent reconciliation failures. Previously some templates used Columns.gap=0 and Paragraph.fontSize 10–11 (rejected by Loops); now all use provider minimums and local validation enforces these limits.

- Updates all nine templates and the reusable base: normalize `Columns.gap` to 12 where 0, and raise any `Paragraph.fontSize` below 12 to 12. Visual hierarchy and copy remain; only minor spacing/text-size shifts.
- Extends validation: `Manifest.Validate()` now recursively checks every `.lmx` file (including unregistered nested bases) for XML well-formedness and explicit provider ranges (`Columns.gap` 12–150, `Paragraph.fontSize` 12–64). Omitted attributes are allowed; `--validate-only` and CI fail on violations.
- Adds tests for boundary values, non-integer attributes, omitted attributes, and nested unregistered files.
- Documents constraints and the MJML-to-LMX translation in the transactional-email skill and Loops README. No dashboard changes or migrations; next reconciliation should pass.

<sup>Written for commit 81ac90dc8e0a12e8c2ca99eecb0d582b156c10a4. Summary will update on new commits.</sup>

<a href="https://cubic.dev/pr/speakeasy-api/gram/pull/5246?utm_source=github" target="_blank" rel="noopener noreferrer" data-no-image-dialog="true"><picture><source media="(prefers-color-scheme: dark)" srcset="https://www.cubic.dev/buttons/review-in-cubic-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://www.cubic.dev/buttons/review-in-cubic-light.svg">Review in cubic</picture></a>

### CODE REVIEW

File: server/internal/email/loopsync/manifest.go [hunk 1]
```diff
@@ -141,19 +141,74 @@ func (m *Manifest) Validate() error {
...
+			}
+		case "Paragraph":
+			if err := validateIntegerAttribute(start, "fontSize", 12, 64); err != nil {
+				return err
+			}
+		}
 	}
 }
 
+func validateIntegerAttribute(element xml.StartElement, name string, minimum, maximum int) error {
```
[the remark this answers is not in the document]
Comment (danielkov) [reply server/internal/email/loopsync/manifest.go:203]: Clarified in 81ac90dc8: omitted attributes intentionally use Loops accepted defaults; explicit values are range-checked. Added a focused omission regression test and updated the skill/README wording.

File: server/internal/email/loopsync/manifest.go [hunk 2]
```diff
@@ -77,6 +78,9 @@ func (m *Manifest) Validate() error {
 	if m.Defaults.FromName == "" || m.Defaults.FromEmail == "" || m.Defaults.ReplyToEmail == "" {
 		return fmt.Errorf("email manifest: sender defaults are incomplete")
 	}
+	if err := validateLMXDirectory(m.Dir); err != nil {
```
[the remark this answers is not in the document]
Comment (danielkov) [reply server/internal/email/loopsync/manifest.go:82]: Fixed in 81ac90dc8: validation now uses a root-scoped recursive walk, with a nested unregistered LMX regression test.

**Review by disintegrator — APPROVED** [review]
